packages/color-kit-block/src/settings.ts

```diff
@@ -11,6 +11,11 @@ export const settings: BlockSettings = {
             columns: 2,
             defaultValue: [],
             showClearAndSelectAllButtons: true,
+            // choices: async ({ appBridge }) => {
```

Comment:
It will change slightly in https://github.com/Frontify/clarify/pull/10882 to have a generic solution

`{ appBridge }` will become `{ getAppBridge }`

Reply:
As PR for that change is still open. I left it as it is.